internalshared/configutil/datadog_sink.go

```diff
@@ -0,0 +1,129 @@
+package configutil
+
+import (
+	"time"
+
+	"github.com/armon/go-metrics"
+	"github.com/armon/go-metrics/datadog"
+	"github.com/mitchellh/cli"
+)
+
+type DatadogSink struct {
+	tags              []string
```

Comment:
You probably need to add a mutex field for getSink to lock to make concurrent use of that method safe.

Reply:
@ncabatoff Done